### ISSUE TITLE
More target="_blank" links in markdown

### ISSUE THREAD

**Issue by pdehaan** [11ty/eleventy#3075]
Hi, first I'm not sure if I should open a new issue as my question is related to that thread, so sorry if I should have done otherwise.

In the project I work on, I have a lot of (text)[link]{attributes} both in my md files body and frontmatter. Links in the body are processed by markdown-it-attrs as expected, but those in the frontmatter seem to ignore the amendment made to the markdown-it lib: links are processed, but not their attributes.

For example: 
.md file
```
---
item: "This is (my frontmatter link)[link]{target='_blank'}"
---
And here is (my body link)[link]{target='_blank'}
```

.njk template file:
```
{{% markdown %}}{{item}}{{% endmarkdown %}}

{{ content | safe }}
```

Result:
```
<p>This is <a href="link">my frontmatter link</a>{target='_blank'}</p>
<p>And here is <a href="link" target='_blank'>my body link</a></p>
```

And my config file:
```
const markdownLib = require('markdown-it')({html: true});
const mdAttrs = require('markdown-it-attrs');
module.exports = function(eleventyConfig) {

  eleventyConfig.setLibrary("md", markdownLib);
  eleventyConfig.amendLibrary("md", markdownLib => markdownLib.use(mdAttrs));

  eleventyConfig.addPairedShortcode("markdown", (content, inline = null) => {
    return inline
      ? markdownLib.renderInline(content)
      : markdownLib.render(content);
  });
};
```

I feel like I'm missing something obvious but can't wrap my head around it.

_Originally posted by @Teufeuleu in https://github.com/11ty/eleventy/issues/2301#issuecomment-1773736042_

**Comment by pdehaan** [comment]
@Teufeuleu I think I have this working in https://github.com/pdehaan/11ty-2301

**Comment by Teufeuleu** [comment]
Thanks! For the record, the solution is simply to add the safe filter (so `{{ item | safe }}` instead of just `{{ item }}`) into the template file...

**Comment by zachleat** [comment]
I believe this one is solved? As the issue was migrated from #2301

> This is an automated message to let you know that a helpful response was posted to your issue and for the health of the repository issue tracker the issue will be closed. This is to help alleviate issues hanging open waiting for a response from the original poster.

> If the response works to solve your problem—great! But if you’re still having problems, do not let the issue’s closing deter you if you have additional questions! Post another comment and we will reopen the issue. Thanks!
